docs/index.rst

```diff
@@ -8,7 +8,7 @@ Introduction
 
 `Hist <https://github.com/scikit-hep/hist>`_ is a powerful Histogramming tool for analysis based on `boost-histogram <https://boost-histogram.readthedocs.io/en/latest/index.html>`_ (the Python binding of the Histogram library in Boost). It is a friendly analysis-focused project that uses `boost-histogram <https://boost-histogram.readthedocs.io/en/latest/index.html>`_ as a backend to do the work, but provides plotting tools, shortcuts, and new ideas.
 
-To get an idea of creating histograms in Hist looks like, you can take a look at the :doc:`Examples <examples/index>`. Once you have a feel for what is involved in using Hist, we recommend you start by following the instructions in :doc:`Installation </development/index>`. Then, go through the :doc:`User Guide </user-guide/index>`, and read the :doc:`Reference </reference/modules>` documentation. We value your contributions and you can follow the instructions in :doc:`Development </development/index>`. Finally, if you’re having problems, please do let us know at our :doc:`Support </support/index>` page.
+To get an idea of creating histograms in Hist looks like, you can take a look at the :doc:`Examples <examples/index>`. Once you have a feel for what is involved in using Hist, we recommend you start by following the instructions in :doc:`Installation <installation>`. Then, go through the :doc:`User Guide </user-guide/index>`, and read the :doc:`Reference </reference/modules>` documentation. We value your contributions and you can follow the instructions in :doc:`Development <development>`. Finally, if you’re having problems, please do let us know at our :doc:`Support <support>` page.
```

Comment:
You taught me something, I didn't know about the `doc:` role! Thanks! Very useful.